File: 231230/시력 검사/eye-test.java
public class Main {
    public static void main(String[] args) {
        // 여기에 코드를 작성해주세요.
        java.util.Scanner sc = new java.util.Scanner(System.in);
        float a = sc.nextFloat();
        float b = sc.nextFloat();
        if(a>=1.0 && b>=1.0){
            System.out.println("High");
        }else if(a>=0.5 && b>=0.5){
            System.out.println("Middle");
            }else{System.out.println("Low");}
    }
}

File: 241123/시력 검사/eye-test.py
left_eye = float(input())
right_eye = float(input())

if left_eye >= 1.0 and right_eye >= 1.0:
    print("High")
elif left_eye >= 0.5 and right_eye >= 0.5:
    print("Middle")
else:
    print("Low")
